packs/spell-effects/spell-effect-fire-shield.json

```diff
@@ -98,6 +98,33 @@
                     "field": "item|system.level.value"
                 }
             },
+            {
+                "alwaysActive": true,
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Fire Shield Block",
+                "option": "water-trait",
+                "predicate": [
+                    {
+                        "or": [
+                            "self:type:character",
+                            "self:type:npc"
+                        ]
+                    }
+                ],
+                "suboptions": [
+                    {
+                        "label": "All traits - Except Water",
+                        "value": "10"
+                    },
+                    {
+                        "label": "Water trait",
+                        "value": "5"
+                    }
+                ],
```

Comment:
This shouldn't be a suboption, since it's just a binary thing. So just a "Defending against a water effect" with a toggle and then two AE likes. This would also need to be localized, but I'd like to see if we could instead find a way to actually automate this part.